doc/REST-interface.md

```diff
@@ -73,7 +73,7 @@ $ curl localhost:18332/rest/getutxos/checkmempool/b2cdfd7b89def827ff8af7cd9bff76
             "reqSigs" : 1,
             "type" : "pubkeyhash",
             "addresses" : [
-               "mi7as51dvLJsizWnTMurtRmrP8hG2m1XvD"
+               "8hVtrkizS3dQbTnPiCVdV7nFuvoce1gJsj"
```

Comment:
that's a p2sh address, not a p2pkh one ;) also see, https://github.com/dashpay/dash/pull/3029#discussion_r304692685 - let's postpone dashification of this file

Reply:
What are you talking about? I see no such commit adding that weird foreign address!